locales/README.md

```diff
@@ -0,0 +1,16 @@
+# Locales
+
+## Note to contributors
+
+Please change only the locale content in the `en-US`
+directory. The other directories are changed using [Pontoon](https://pontoon.mozilla.org/projects/firefox-profiler/).
+
+Here are some simple rules:
+1. If you change the string in a way that changes the meaning, you need to
+   change the string key as well. If that doesn't change the meaning (for
+   example, if that's a typo fix), then the string key will stay the same.
+2. If you include a variable, make sure to add a comment explaining it.
+3. If the context for a string isn't obvious, please add a comment explaining it too.
+
+## Note to translators
+Please use [Pontoon](https://pontoon.mozilla.org/projects/firefox-profiler/) to update languages other than `en-US`.
```

Comment:
nit: empty line after header.

Should we add that pull requests used to update localizations will be closed?

Reply:
I added `If a pull request updates those, we'll ask you to revert these changes.`